pkg/cluster/cluster.go

```diff
@@ -378,6 +384,34 @@ func (c *Cluster) deleteClientServiceLB() error {
 	return nil
 }
 
+func (c *Cluster) deleteUnusedServices() error {
+	s := labels.SelectorFromSet(map[string]string{
+		"etcd_cluster": c.Name,
+		"app":          "etcd",
+	})
+	r, err := labels.NewRequirement("etcd_node", selection.Exists, nil)
+	if err != nil {
+		return err
+	}
+
+	svcs, err := c.KubeCli.Services(c.Namespace).List(k8sapi.ListOptions{
```

Comment:
This func get and delete all instances' service(func name may be bad). Its fine because I call this func at deleting cluster stage.